components/card/skin.css

```diff
@@ -31,7 +31,7 @@ governing permissions and limitations under the License.
   }
 
   .spectrum-Card-title {
-    color: var(--spectrum-alias-text-color);
+    color: var(--spectrum-global-color-gray-900);
```

Comment:
Any reason for this? I'd rather see an alias than a global, and it seems that maybe this was changed? I see

```
  --spectrum-alias-text-color: var(--spectrum-global-color-gray-800);
  --spectrum-alias-text-color-hover: var(--spectrum-global-color-gray-900);
```

Reply:
Perhaps this should be `gray-800`?

Reply:
Sorry, I should have removed that as well, one sec, and I'll update title and subtitle to just use the Heading and Detail components and delete those.

Reply:
@GarthDB I just removed it.

Reply:
@jianliao beat me to it.